target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-fritzrepeater-1200.dts

```diff
@@ -243,6 +237,30 @@
 	status = "okay";
 };
 
+&ethphy0 {
+	status = "disabled";
+};
+
```

Comment:
Are we sure we want to disable ethphy0? What is left? There is 1 port on this device, not 0.